android/build.gradle

```diff
@@ -9,7 +9,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.2'
+        classpath 'com.android.tools.build:gradle:4.0.1'
```

Comment:
Can you extract this update, this one https://github.com/RevenueCat/purchases-flutter/pull/193/files#diff-90156c7b6ee416a97411a15877e2e1eac7b5c7c630ead570c421caf5153362d4R8 and https://github.com/RevenueCat/purchases-flutter/pull/193/files#diff-38561c91b2badbeee265859a611c25856b5e46a13aaf19e30d2adb6f859b9745R6 to a separate PR? That way changes are more isolated since this is not really part of the `canMakePayments` support and if they cause issues, it's easier to revert and spot what's causing problems.

Reply:
I believe this part is needed for tests to pass locally, though, right? 
Agreed with separating it as a PR, but I definitely love that we're finally taking care of this. I've been updating locally and then not committing the changes for a while, which is not a great way to do things.

Reply:
I also confirmed yesterday and it looks like this is the default when creating a new Flutter plugin. So yes, thanks for updating these 🙌🏻